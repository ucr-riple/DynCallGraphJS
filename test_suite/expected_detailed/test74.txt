{
    'global (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test74.js:21:1:21:7)': [
        'main (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test74.js:10:1:13:2)'
    ],
    'main (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test74.js:11:22:11:34)': [
        'act (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test74.js:14:1:20:2)'
    ],
    'act (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test74.js:16:9:18:11)': [
        'forEach (Native)(/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test74.js:16:9:18:11)'
    ],
    'forEach (Native) (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test74.js:16:9:18:11)': [
        'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test74.js:16:19:18:10)'
    ],
    'global (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test74.js:22:1:22:14)': [
        'bar (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test74.js:4:1:6:2)'
    ]
}