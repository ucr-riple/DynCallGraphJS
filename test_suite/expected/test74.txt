{ '(test74.js:21:1:21:7)':
   [ '(test74.js:10:1:13:2)' ],
  '(test74.js:11:22:11:34)':
   [ '(test74.js:14:1:20:2)' ],
  '(test74.js:16:9:18:11)': [ 'forEach (Native)' ],
  'forEach (Native)':
   [ '(test74.js:16:19:18:10)' ],
  '(test74.js:22:1:22:14)':
   [ '(test74.js:4:1:6:2)' ] }